spec/models/generic_file_spec.rb

```diff
@@ -76,8 +76,10 @@
     end
 
     it "should have attached content" do
-      subject.add_file(File.open(fixture_path + '/world.png'), path: 'content', original_name: 'world.png')
-      expect(subject.content).to be_kind_of ActiveFedora::File
+      subject.apply_depositor_metadata('jcoyne')
+      subject.save
```

Comment:
The subject needs to be saved before the service will work.  To save the file I needed to apply depositor Metadata.